l10n_br_fiscal/models/document_fiscal_mixin_methods.py

```diff
@@ -28,6 +28,14 @@ def _get_amount_lines(self):
         """Get object lines instaces used to compute fields"""
         return self.mapped("line_ids")
 
+    def is_sale_industry(self):
```

Comment:
Essa parte que define se é uma operação de industria ou comercio deveria estar no mapeamento dos impostos no fiscal.operation.line, lá se a empresa for do simples nacional deveria preencher o icmssn_range_id